apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryClient.Queries.cs

```diff
@@ -174,5 +176,180 @@ public virtual BigqueryQueryJob PollQueryUntilCompleted(JobReference jobReferenc
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>The query job created. Use <see cref="GetQueryJobAsync(JobReference,GetQueryResultsOptions,CancellationToken)"/> to retrieve
+        /// the results of the query.</returns>
+        public virtual Task<BigqueryJob> CreateQueryJobAsync(string sql, CreateQueryJobOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Asynchronously creates a job for a query/command, with more facilities than <see cref="ExecuteQueryAsync(BigqueryCommand, ExecuteQueryOptions,CancellationToken)"/>,
+        /// including the option to store the results in a persistent table. This overload allows query parameterization, and is preferred over
+        /// <see cref="CreateQueryJobAsync(string, CreateQueryJobOptions,CancellationToken)"/> when values need to be passed in.
+        /// </summary>
+        /// <param name="command">The command to execute. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>The query job created. Use <see cref="GetQueryJobAsync(JobReference,GetQueryResultsOptions,CancellationToken)"/> to retrieve
+        /// the results of the query.</returns>
+        public virtual Task<BigqueryJob> CreateQueryJobAsync(BigqueryCommand command, CreateQueryJobOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Asynchronously retrieves the results of the query job with specified by project ID and job ID.
+        /// This method just creates a <see cref="JobReference"/> and delegates to <see cref="GetQueryJobAsync(JobReference, GetQueryResultsOptions,CancellationToken)"/>.
+        /// </summary>
+        /// <param name="jobId">The job ID. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>A task representing the asynchronous operation. When complete, the result is
+        /// the <see cref="BigqueryQueryJob"/> representing the query.</returns>
+        public virtual Task<BigqueryQueryJob> GetQueryResultsAsync(string jobId, GetQueryResultsOptions options = null, CancellationToken cancellationToken = default(CancellationToken)) =>
```

Comment:
No, that was a mistake. But we have a bit of a problem in that CreateQueryJob and GetQueryJob return different types - it sort of makes sense, but only once you think about it for a while.

Have renamed them all to GetQueryResults for now, as that's the underlying API name. Will need to think about it again when we rename BigqueryQueryJob to something better.